{{cookiecutter.app_name}}_role/.yamllint

```diff
@@ -3,9 +3,7 @@
 extends: default
 
 ignore: |
-  .travis.yml
-  molecule/
-  tests/
+  molecule/default/tests/
```

Comment:
Should we ignore other scenarios besides the default one?

Reply:
I agree, does glob expresions works? Maybe molecule/**/tests is a proper expresion.